local-setup/README.md

```diff
@@ -0,0 +1,36 @@
+#  How to use the local-setup
+
+## Prerequisite
+- worker built with `make`
+- integritee-node built with `cargo build --release`
+
+## Steps
+Adapt or create your own config file, as in the example of `simple-config.json`. Be mindful of the ports in case you're running the script on a server multiple people are working on.
+
+### Launch worker and node in terminal one
+You can launch the workers and the node with:
+```bash
+cd <worker directory>
+./local-setup/launch.py ./local-setup/simple-config.json
+```
+wait a little until all workers have been launched. You can stop the worker and node simply by pressing `Ctrl + c`.
+
+### Open a second terminal to show logs
+```bash
+cd <worker directory>/local-setup
+# If you work with docker: exec into the running container:
+docker exec -it [container-id] bash
+cd work
+# run the tmux logger script with:
+./tmux_logger.sh
+```
+
+You can remove the tmux session of the script by running
+```bash
+tmux kill-session -t integritee_logger
+```
```

Comment:
Do you use that sometimes? I just exit tmux, and it kills the session automatically.

Reply:
Does it? I remember it not being killed automatically. But it's been months ago since I tested it..

Reply:
Well, then let's keep it. It does not hurt.